tests/containers/podman_network_cni.pm

```diff
@@ -52,10 +52,36 @@ sub run() {
         assert_script_run('podman network inspect newnet1 --format "{{range .Subnets}}Subnet: {{.Subnet}} Gateway: {{.Gateway}}{{end}}"');
     }
 
+    #connect, disconnect & reload
+    unless ($old_podman) {
+        record_info('Prepare', 'Prepare three containers');
+        assert_script_run('podman run -id --rm --name container1 -p 1234:1234 registry.opensuse.org/opensuse/tumbleweed');
```

Comment:
from my experience with pulling containers, I would first run the command `podman pull registry.opensuse.org/opensuse/tumbleweed` using `script_retry` before using the image. 
Example: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/bci_repo.pm#L82

Reply:
@jlausuch Your suggestions was implemented.